service/storage_service.cc

```diff
@@ -1897,8 +1897,16 @@ class topology_coordinator {
         slogger.info("raft topology: coordinator fiber found a node to work on id={} state={}", node.id, node.rs->state);
 
         switch (node.rs->state) {
-            case node_state::normal:
-            case node_state::none: {
+            case node_state::none:
+                if (_topo_sm._topology.normal_nodes.empty() && _topo_sm._topology.transition_nodes.empty()) {
```

Comment:
Is `transition_nodes` check is needed. How can we process a node in state `none` while there is another node that is in transition?

Reply:
Remove please. It confused me a little.

Reply:
Removed.